source/import-export.txt

```diff
@@ -158,6 +160,48 @@ Procedure
       .. include:: /includes/steps/export-filter.rst
 
       .. include:: /includes/fact-export-progress-bar.rst
+   
+   .. tab:: Export Aggregation Results
+      :tabid: export-Aggregation
+
+      To export results from your aggregation pipeline:
```

Comment:
Do you think it's worth having a reference to how to export the results of a pipeline also in the aggregation builder docs? Given that we are showing the `Export` button the UI we may want to say what it does.